source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,76 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the backup option you use for the site.
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync.
+If you suffer complete site loss with site replication enabled, you can restore IAM policies, objects, metadata, and other :ref:`replicable data <minio-site-replication-what-replicates>` from the remaining site(s).
+
+1. Deploy a new MinIO site using the same ``root`` credentials as used on other deployments in the site replication configuration
+2. Configure the new site with the same Identity Provider (IDp) as the other sites
+3. :ref:`Expand the existing site replication <minio-expand-site-replication>` by adding the newly deployed site
+4. Remove the lost site from the site replication configuration
+
+Site replication healing automatically adds the buckets and objects from the existing site(s) to the new site with no further action.
+
+Replication Resync
+------------------
+
+For scenarios where :ref:`bucket replication <minio-bucket-replication>` was in place prior to the failure, you can use :mc-cmd:`mc replicate resync` to restore data to a new site.
+Create a new site to replace the failed deployment, then synchronize the data from an existing bucket to the new site.
+
+1. Deploy a new MinIO site
+2. Set up IAM and users as needed
+3. On the site with data, create a new ``remote target`` using the :mc-cmd:`mc admin bucket remote add` command and record the ARN from the output
+4. From the site with the data, use the :mc-cmd:`mc replicate resync` command with the ARN from the previous command to rebuild the on the new site
+5. Wait for re-synchronization to complete (us :mc-cmd:`mc replicate resync status` to check)
+6. Set up bucket replication rule(s) from the new MinIO site to the existing target bucket(s)
+7. `(Optional)` Delete the bucket replication rules from the target deployment(s) to restore an active-passive replication scenario
+
+Bucket Replication
+------------------
+
+Alternatively, use :ref:`bucket replication <minio-bucket-replication>` directly to restore the site contents by performing a replication from the target bucket(s) to a new MinIO site.
```

Comment:
Weird to start with Alternatively since this is a section header. Alternatively to what? We lose the context here.

Reply:
It's an alternative to the resync process above. Removed.